src/Rector/Contrib/PHPUnit/SpecificMethodRector.php

```diff
@@ -67,14 +68,12 @@ public function isCandidate(Node $node): bool
         }
 
         $firstArgumentValue = $node->args[0]->value;
-        if (! $firstArgumentValue instanceof FuncCall) {
+
+        $funcCallName = $this->resolveFunctionName($firstArgumentValue);
```

Comment:
Here everything that is not `FuncCall` node is skipped. So `empty()` was too, since it's `Empty_` node.